upkie/cpp/actuation/BulletInterface.cpp

```diff
@@ -225,6 +225,7 @@ void BulletInterface::observe(Dictionary& observation) const {
     sim("contact")(link_name)("num_contact_points") =
         contact_data_.at(link_name).num_contact_points;
   }
+  sim("env_collision") = environment_collision_;
```

Comment:
Using the same name in logs and code can make our life a (tiny) notch easier (one difference less to remember :wink:)